src/components/VFRChartsList.js

```diff
@@ -54,9 +57,9 @@ class VFRChartsList extends Component {
 
 const styles = StyleSheet.create({
   container: {
-    alignItems: 'center',
-    backgroundColor: Colors.secondary,
     flex: 1,
+    alignItems: 'stretch',
+    backgroundColor: 'green',
```

Comment:
Just checked it.  I think because ChartCell has a background color we dont even need this line